main.go

```diff
@@ -222,13 +223,30 @@ func main() {
 	result, platformsDetected := scanner.GenerateScanResult(searchDir, hasSSHKey)
 
 	// Upload results
-	if resultClient != nil {
-		log.TInfof("Submitting results...")
-		if err := resultClient.uploadResults(result); err != nil {
-			failf("Could not send back results: %s", err)
+	var resultBytes []byte
```

Comment:
This variable is not used outside of the if statement below so I think it can be moved from here to there.

Reply:
Updated in https://github.com/bitrise-steplib/steps-project-scanner/pull/148/commits/bdcb0dae4a42cf642571062fc889b7db71efd8b4